src/extensions/yfm/Math/index.ts

```diff
@@ -38,7 +38,7 @@ export const Math: ExtensionAuto<MathOptions> = (builder, opts) => {
     builder.addKeymap(() => ({
         Enter: ignoreIfCursorInsideMathInline, // ignore breaks in math inline
         Backspace: chainCommands(
-            moveCursorToEndOfMathInline,
+            moveCursorLeftOfMathInline,
             removeEmptyMathInlineIfCursorIsAtBeginning,
```

Comment:
Now you need to swap these two commands, because the empty math_inline block is not being deleted now